gulp/config/plugins.js

```diff
@@ -1,14 +1,12 @@
 import plumber from "gulp-plumber"; // handling of errors
 import notify from "gulp-notify"; // tooltip messages
 import browsersync from "browser-sync"; //local server
-import newer from "gulp-newer"; //checking of update
```

Comment:
these changes must be in gulp-implementation branch, roll them back, please

Reply:
Done